features/step_definitions/oauth.rb

```diff
@@ -47,7 +47,7 @@
 Given /^authentication successfully rolls out after config changes$/ do
   ensure_admin_tagged
   interval_time = 5
-  timeout = 400 # set 400 seconds here due to https://bugzilla.redhat.com/show_bug.cgi?id=1958198, after the bug fixed, the seconds should be reduced accordingly
+  timeout = 900 # set 900 seconds here due to https://bugzilla.redhat.com/show_bug.cgi?id=1958198, after the bug fixed, the seconds should be reduced accordingly
   stats = {}
   error = nil
   step %Q/operator "authentication" becomes progressing within #{timeout} seconds/
```

Comment:
Could you separate timeout settings for `becomes progressing` and `becomes available/non-progressing/non-degraded`?

Reply:
@xingxingxia updated, please help review again , thanks.